manager/testdata/notebook/read-flow-np/setup.sh

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env bash
```

Comment:
Is it possible to use `testdata/notebook/read-flow/setup.sh' instead of creating a new setup.sh file to avoid duplication? the tls tests also run this script and use ifdefs to apply additional resources if needed

Reply:
Fixed.